app/scripts/metamask-controller.js

```diff
@@ -1472,10 +1472,14 @@ export default class MetamaskController extends EventEmitter {
       getMetaMetricsState: () =>
         this.metaMetricsController.state.participateInMetaMetrics ?? false,
       state: initState.UserStorageController,
+      env: {
+        isAccountSyncingEnabled: true,
```

Comment:
Can't wait!!

(no real opinion on this). Was thinking we have this turned off on this PR, and then we can make a subsequent PR to turn on this feature.

But we can keep as is otherwise (just notify extension team so QA doesn't get nicely surprised)